.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Recipe
+about: This template makes sure relevant information is contained in a new recipe pull request.
+title: [Dataset Name]
+labels: ''
+assignees: ''
+
+---
+<!-- 
+👋 Thanks for opening a PR! This template describes the relevant information for a new recipe pull request.
+
+Appropriate files:
+Do you have a meta.yaml and a recipe.py in this PR? If you don't know what those are, please check out the sandbox https://mybinder.org/v2/gh/pangeo-forge/sandbox/binder or https://pangeo-forge.readthedocs.io/en/latest/introduction_tutorial/intro_tutorial_part3.html
```

Comment:
A subtle but perhaps important point: we don't require the recipe module to be named `recipe.py` (even though this is what the example file in the sandbox is called). Contributors can call the recipe module whatever they'd like; they then specify the module name (and the name of the recipe class instance within it) via the `meta.yaml`'s recipe section:

https://github.com/pangeo-forge/roadmap/blob/master/doc/adr/0002-use-meta-yaml-to-track-feedstock-metadata.md#recipes-section

Also, I think we should probably link to the static version of these example files? Something like 

https://github.com/pangeo-forge/sandbox/tree/main/recipe

rather than the binder link which will take longer to load?

Reply:
Thanks for the clarification!  I added both the static link and the binder env